packages/core/src/errors/error-reporter.ts

```diff
@@ -255,7 +255,9 @@ export class ErrorReporter {
 		if (tags) event.tags = { ...event.tags, ...tags };
 	}
 
-	private async getEventLoopBlockIntegration() {
+	private async getEventLoopBlockIntegration(serverType: string) {
+		if (serverType === 'task_runner') return []; // user code blocking the event loop is not actionable
```

Comment:
Instead of deciding this here based on the server type, I think it would be cleaner if the `init` method had a parameter that tells whether to initialize this as well. That way it's more decoupled